lib/wellies_web/temperature/current.ex

```diff
@@ -0,0 +1,31 @@
+defmodule ForecastNow do
+  defstruct [:date, :hour, :temperature]
+end
+
+defmodule WelliesWeb.CurrentTemperature do
+  alias WelliesWeb.ResponseHandler
+  alias WelliesWeb.OpenWeatherParser
+  alias WelliesWeb.OpenWeatherApi
+
+  def request_current_forecast(id) do
```

Comment:
If the module is called `CurrentTemperature`, you could just call the function `request_forecast`. Nonetheless, like I mentioned before, it could make sense to call the module `CurrentForecast` (since it's not just temperature, but also time?), and then the function `request` simply. Not sure, they're just some ideas.